demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -763,3 +770,41 @@ def test_get_unfiltered_changed_files_from_git_in_external_pr_use_case(
     )
     output = initializer.get_unfiltered_changed_files_from_git()
     assert output[0] == expected_output
+
+
+def test_check_metadata_version_bump_on_content_changes(mocker, repo):
+    """
+    Given: pack with newly added integration.
+    When: Initializing ValidateManager using git
+    Then: Ensure PackMetadataVersionShouldBeRaisedValidator is initialized and the external args are properly passed.
+    """
+    pack = create_pack_object(["currentVersion"], ["1.1.1"])
+    integration = create_integration_object()
+    pack.content_items.integration.extend(integration)
+    validation_results = ResultWriter()
+    config_reader = ConfigReader(category_to_run="use_git")
```

Comment:
you can use the new argument **specific_validations** here.
validation_config shouldn't be updated in any case.